### PR TITLE
add Nerima Ortho imagery, Tokyo Japan

### PR DESCRIPTION
**Pull request by nyampire**
I would like to add an Orthoimagery resources to the repository.

The information wiki page is [on OSM wiki](https://wiki.openstreetmap.org/wiki/Nerima_ortho).

If this could be valid to merge, I would like to make more resources in Japan region.

This is the first time to add, so please forgive me if it has some errors. I've tested at [Imagery index](https://ideditor.github.io/imagery-index/index.html) and got valid result.

### CODE REVIEW

**Comment by cicku** [comment]
The [license page](https://www.city.nerima.tokyo.jp/kusei/tokei/opendata/opendatasite/tokei_kusei/koku-shashin.html) shows that the data is CC BY 4.0.

Per OSMF policy, you have to obtain a waiver or permission on the usage:

- https://wiki.osmfoundation.org/wiki/Licence/Licence_Compatibility#CC_BY
- https://blog.openstreetmap.org/2017/03/17/use-of-cc-by-data/

I've seen that you have obtained a permission but could not see anything proving that, would you please upload the permission (email or document or screenshot) to OSM wiki?

**Comment by nyampire** [comment]
Yes, it is under CC-BY 4.0. So I already got permission/waiver from Publisher, and it is summarized in OSM wiki text.

> ライセンス
> CC BY 4.0 ライセンスです。
> また、練馬区に対し、user:nyampireがOpenStreetMapでのデータ利用についてメールで申請し、許諾をいただいています。 

(Deepl translation)
> License.
> CC BY 4.0 license.
> In addition, user:nyampire has applied to Nerima City for permission to use the data on OpenStreetMap via email.

**Comment by andrewharvey** [comment]
> In addition, user: nyampire has applied to Nerima Ward by e-mail regarding the use of data on OpenStreetMap and has obtained permission.

Can their email response which granted permission be included on the wiki to document this?

**Comment by andrewharvey** [comment]
The completed waiver form from LWG at https://drive.google.com/file/d/0B3PN5zfbzThqeTdWR1l3SzJVcTg/view?resourcekey=0-PzVtHArfxvbYidpW2-AVTg is the gold standard, if that could be completed and uploaded to the wiki that's ideal.

**Comment by nyampire** [comment]
@andrewharvey 
> wiki documentation

Yes, it includes that. full text is following. Please use deepl to translate.
It includes `wiki documentation for attribution` and `conflict between CC-BY and ODbL`.

### My mail
```
■利用目的
OpenStreetMapでこの航空写真をトレース（なぞり書き）し、二次著作物として、地図データの作成を行います。
作成した地図データは、ODbLというオープンなライセンスで配布されます。

https://opendatacommons.org/licenses/odbl/

■利用方法
公開されているデータを利用し、画像配信のサーバを個人で構築しました。
データの利用は私のサーバを経由しますので、作業にあたって練馬区様のサーバに負荷を与えることはありません。

■連絡事項
1. いただいたデータのCC-BYの帰属表示ですが、以下の2つの方法で行います。

* OpenStreetMapの編集作業へのメタデータ付与
* ドキュメント用ページ (https://wiki.openstreetmap.org/wiki/Nerima_ortho) への記載

2. CC-BYデータを元とした二次著作物を、ODbLライセンスで配布するにあたって細いことで申し訳ありませんが、ODbLとCC-BYには少しだけ差異がありますので、そのご説明をさせてください。

* ODbLでは、データを利用した二次著作物へのコピーライト・プロテクションを行うことができます。
* CC-BYでは、データを利用した二次著作物へプロテクションをかけることを禁じています。

ついては、OpenStreetMapおよびその利用者が、航空写真データを利用するにあたって、二次著作物にコピーライト・プロテクションをかける可能性があることをお許しください。
なお、これによって、練馬区からのデータの配布やそこで掲げられている利用規約・免責事項などに影響が及ぶことはありません。
```

### response
```
練馬区情報政策課のXXと申します。

先ほどのお問合せの件について、ご回答いたします。

区では、保有する情報を自由にご活用いただけるよう、オープンデータの公開に取り組んでいます。
このため、OpenStreetMapでデータをご利用いただくことについても、特に差し支えはありません。
```

And sure, it is better to use the template from LWG([I was a member there](https://wiki.osmfoundation.org/wiki/Licensing_Working_Group)), but I think it is more better to use the summary text to ask the city officers in JP. (they are non-specialist for ODbL, and as it is quite tough tasks to get their sign or stamp)

I'll copy and paste above messages on OSM wiki if needed.

**Comment by cicku** [comment]
I’d say yes on this. Thank you for all efforts you have made! ✅

File: sources/asia/jp/Tokyo-Nerima-orthophoto.geojson [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            "required": true
+        },
+        "name": "Tokyo Nerima-ku Imagery 2016",
+        "url": "http://nyampire.conohawing.com/ortho-nerima-ku/{zoom}/{x}/{-y}.png",
+        "min_zoom": 12,
+        "max_zoom": 19,
+        "license_url": "https://wiki.openstreetmap.org/wiki/Nerima_ortho",
+        "country_code": "JP",
+        "type": "tms",
+        "available_projections": "EPSG:4326",
```
Comment (rbuffat) [inline sources/asia/jp/Tokyo-Nerima-orthophoto.geojson]: `"available_projections": "EPSG:4326",` is not needed for `"type": "tms"`

**Comment by rbuffat** [comment]
@nyampire Thanks for this PR and sorry for not having time to look at it earlier!  `available_projections` is not needed for `tms`. Additionally, if the server has a privacy policy, it can be added with `privacy_policy_url.`. The privacy policy URL typically contains information about what information the server saves about the user (cookies, logs, data) etc.  If there is no privacy policy available, this is also no problem. The key can just be omitted. The checks will fail, but the PR can be merged nevertheless.

**Comment by nyampire** [comment]
@cicku @rbuffat Thank you!

I've tried to delete the EPSG line. Please check https://github.com/osmlab/editor-layer-index/pull/1285/commits/5374747412d8b3ba6675ac4537c02bb13e0e0d78.

And the server does not have `privacy policy`. It is just a simple rental server (storage).

**Comment by rbuffat** [comment]
Thanks @nyampire !

**Pull request merged**